README.md

```diff
@@ -17,20 +17,25 @@
 3. Make sure speakers are still available and have their slides ready
 4. Send reminders to people as time draws near
 5. (Bonus) Uploading presenter notes to [hackmd.io](https://hackmd.io/cLr7SN-ER2uOvYc_IlC87A)
+6. Double check audio / video quality for folks recording
 
 ## Production
 
 1. Drop into the world, decorate, wait for guests
 2. Start recording/streaming to twitch or youtube
+  - Use [OBS](https://obsproject.com/) for the recording.
+  - Have camera people right click slides to enlarge them periodically
 3. Moderate talks to stay on schedule
```

Comment:
Someone should have this as a designated job.

Reply:
Added!